listenbrainz/webserver/static/css/theme/boostrap/modals.less

```diff
@@ -97,6 +97,55 @@
 .modal-body {
   position: relative;
   padding: @modal-inner-padding;
+
+  #dropdown-container {
+    padding-bottom: 10px;
+  }
+
+  .rating-container{
+    line-height: 30px;
+    .rating-stars{
+      > span{
+        position:relative;
+        top:4px;
+      }
+    }
+  }
+}
+
+#CBReviewModal {
+
+  .dropdown-toggle{
+    font-weight: bold;
+    .caret{
+      margin-left: 6px;
+    }
+  }
+
+  .dropdown-menu {
+    min-width: 140px;
+    border-radius: 4px;
+    padding: 10px 0px;
+    width: 250px;
+    color: @hate;
```

Comment:
That doesn't look right, that `hate` variable is used for ratings but we shouldn't rely on it always being there.
But I do see the definition of @hate is just `@hate: @blue` . @blue is defined in static/css/colors.less and those aren't going anywhere, so you can use @blue directly.